js/main.js

```diff
@@ -6,35 +6,31 @@ const time = document.getElementById("time"),
   focus = document.getElementById("focus");
 
 // Show time
-function showTime() {
+let showTime = () => {
```

Comment:
Since we are not reassign this variable, there is no reason to use let here.
Please, use "cost" instead.
Review js style guide for best practices: https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations

Reply:
Thank you for your comment. I've replaced let with const in all arrow functions.